tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -781,10 +784,7 @@ public int doCalc(boolean warmup, int run) {
                 if (querySettings.alternative)
                     req.setAlgorithm(ALT_ROUTE);
 
-                if (querySettings.withInstructions)
-                    req.setPathDetails(Arrays.asList(Parameters.Details.AVERAGE_SPEED));
-
-                if (querySettings.simplify) {
+                if (querySettings.simplifyAndPD) {
                     req.setPathDetails(Arrays.asList(Parameters.Details.AVERAGE_SPEED, Parameters.Details.EDGE_ID, Parameters.Details.STREET_NAME));
```

Comment:
Here we enable simplify by adding path details? Do we still have a measurement with instructions and path details?

Reply:
I think "simplification" is always done for "forServer" (used here in Measurement, see line 182), except if `!simplify`, then WAY_POINT_MAX_DISTANCE=0

Reply:
Aha :) It seems confusing that `setPathDetails` has anything to do with instructions?

Reply:
Do we really need `forDesktop/Server/Mobile`? It happened to me at least twice that I wanted to disable/enable path simplification and then I found myself calling `forServer` or `forDesktop` to achieve this 'indirectly'...

Reply:
> Aha :) It seems confusing that setPathDetails has anything to do with instructions?

Will introduce a separate pd=true/false variable

> Do we really need forDesktop/Server/Mobile? It happened to me at least twice that I wanted to disable/enable path simplification and then I found myself calling forServer or forDesktop to achieve this 'indirectly'...

You mean exposing setSimplifyResponse and removing forXY?

Reply:
> It happened to me at least twice that I wanted to disable/enable path simplification and then I found myself calling forServer or forDesktop to achieve this 'indirectly'...

Hmmh, looking into this ... and "forServer" (i.e. with simplification) should be the default. Also you could call `hopper.getRouterConfig().setSimplifyResponse` to change this.

Reply:
> You mean exposing setSimplifyResponse and removing forXY?

Yes if thats possible? Or was it meant similar to `ch.disablingAllowed` (make sure users cannot disable path simplification)?

> Hmmh, looking into this ... and "forServer" (i.e. with simplification) should be the default. Also you could call hopper.getRouterConfig().setSimplifyResponse to change this.

Ah ok if we can change simplify via the router config thats good (I thought the last time I wanted to do this it was not possible).

Reply:
> Yes if thats possible? Or was it meant similar to ch.disablingAllowed (make sure users cannot disable path simplification)?

It is meant as the default and for desktop it doesn't make much sense as we can transmit the full and original path, but client bandwidth or client power can be limited and it is usually faster to compress the response even if it takes time. E.g. komoot returns a custom format probably unsimplified resulting in ~5MB (just picked a 200km route) where our route returns quickly and only needs <100KB :) (but probably this is more due to our compact path details representation and not so much due to the simplification. maybe interesting improvement @jtheuer ;) )